src/functions/template-tags/html.php

```diff
@@ -127,8 +127,79 @@ function tribe_disabled( $disabled, $echo = true ) {
 
 		if ( ! $echo ) {
 			return $disabled_string;
-		} else {
-			echo $disabled_string;
 		}
+
+		echo $disabled_string;
+	}
+}
+
+/**
+ * Adds rel attribute for links.
+ *
+ * @since TBD
+ *
+ * @param string|array $rel  Value(s) for rel attribute.
+ * @param boolean      $echo Whether to echo the string or return it.
+ *
+ * @return string The rel attribute.
+ */
+function tribe_rel( $rel, $echo = true ) {
+	if ( empty( $rel ) ) {
+		return;
+	}
+
+	if ( is_array( $rel )) {
+		$rel = implode( ' ', $rel );
+	}
+
+	$attr = 'rel="' . esc_attr( $rel ) . '"';
+
+	if ( ! $echo ) {
+		return $attr;
+	}
+
+	echo $attr;
+}
+
+/**
+ * Adds target attribute for links.
+ * If target is '_blank' also adds appropriate rel attributes.
+ *
+ * @since TBD
+ *
+ * @param string       $target Value for the target attribute.
+ * @param array|string $rel    Value(s) for rel attribute.
+ * @param boolean      $echo   Whether to echo the string or return it.
+ *
+ * @return string The target attribute. Potentially also the rel attribute.
+ */
+function tribe_target( $target, $rel = [], $echo = true ) {
```

Comment:
* :no_entry_sign: Function name "tribe_target" is not in camel caps format
* :no_entry_sign: TRUE, FALSE and NULL must be uppercase; expected "TRUE" but found "true"
* :no_entry_sign: Opening brace should be on a new line

<sub>hash: f861880e41fdad49d11ba087ed336a45</sub>